decred/decred/dcr/dcrdata.py

```diff
@@ -639,8 +639,6 @@ def tx(self, txid):
             try:
                 # Grab the hex encoded transaction
                 txHex = self.dcrdata.tx.hex(txid)
-                if not txHex:
-                    raise DecredError("failed to retrieve tx hex from dcrdata")
                 msgTx = msgtx.MsgTx.deserialize(ByteArray(txHex))
```

Comment:
`txHex` is never `False`: either the call succeeds returning a valid value or it raises an Exception, which is handled in this block.